### PR TITLE
Add an example of Shared elements that are seekable using SeekableTransitionState

### PR DESCRIPTION
**Pull request by riggaroo**
Added an example of seekable transitions, was previously approved by Doris but pull request got outdated. #273

### CODE REVIEW

**Review by mlykotom — COMMENTED** [review]
should we add `android:enableOnBackInvokedCallback="true"` to the android manifest? otherwise the predictive back doesn't work (I don't know if we don't break some other snippet though)

**Comment by mlykotom** [comment]
When I cancel the back gesture, then it doesn't work again :(

https://github.com/user-attachments/assets/86cd86f1-b4b7-4604-83d5-1f6a3faa1896

File: compose/snippets/src/main/java/com/example/compose/snippets/animations/sharedelement/CustomizeSharedElementsSnippets.kt [hunk 1]
```diff
@@ -628,3 +636,91 @@ fun PlaceholderSizeAnimated_Demo() {
...
+                } catch (e: CancellationException) {
+                    // ignore the cancellation
+                }
+            }
+            // code for completion
+            seekableTransitionState.animateTo(seekableTransitionState.targetState)
+        } catch (e: CancellationException) {
+            // code for cancellation
+            seekableTransitionState.animateTo(seekableTransitionState.currentState)
+        }
```
Comment (mlykotom) [inline compose/snippets/src/main/java/com/example/compose/snippets/animations/sharedelement/CustomizeSharedElementsSnippets.kt]: I find it slightly confusing why do I need to suppress the cancellation exceptions here.
Can we explain it a bit more in the comments? 🙏🏼

Comment (riggaroo) [reply compose/snippets/src/main/java/com/example/compose/snippets/animations/sharedelement/CustomizeSharedElementsSnippets.kt]: Added comments.

**Comment by riggaroo** [comment]
> When I cancel the back gesture, then it doesn't work again :(
> 
>  broken.mp4

Fixed this by changing to snapTo instead of animateTo in the cancellation block, could you try it again?

**Review by mlykotom — APPROVED** [review]
Thank you, so much clearer with the comments. I tried it on the phone and it's working ❤️

**Pull request merged**